src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -64,6 +68,17 @@ public function execute(array $input, array $options = array())
             $input = array_merge(array('command' => $this->command->getName()), $input);
         }
 
+        if ($this->inputs) {
+            if (null === $this->command->getHelperSet()) {
+                $helper = new QuestionHelper();
+                $this->command->setHelperSet(new HelperSet(array($helper)));
+            } else {
+                $helper = $this->command->getHelper('question');
+            }
+
+            $helper->setInputStream($this->createInputStream());
+        }
```

Comment:
This looks wrong to me. Here we are only dealing with one specific case, which is the question helper. We should rather find a way to make this "generic" if possible.